docs/en/compute/Spark-On-Kubernetes.md

```diff
@@ -145,9 +140,19 @@ $ kubectl create clusterrolebinding spark-role --clusterrole=edit \
 
 #### Submit a Spark job
 
+The following command runs an example word count job in the Alluxio location
+`/LICENSE`. Please ensure this file exists in your Alluxio cluster, or otherwise
+change the path to a file which does exist.
+
+The output and time taken can be seen in the logs for Spark driver pod. Refer to
+the Spark [documentation](https://spark.apache.org/docs/latest/running-on-kubernetes.html)
+for further details about running Spark on Kubernetes. For example, you may find
+additional details on some of the flags used in this command
+[here](https://spark.apache.org/docs/latest/running-on-kubernetes.html?q=cluster-info#cluster-mode).
+
 Run the job from the Spark distribution directory
 ```console
-$ ./bin/spark-submit --master k8s://https://<kubernetes-api-server>:8443 \
```

Comment:
nit: Yea it is indeed 6443 now but I'm not sure why this changed. If you happen to figure out could you let me know?
One possibility is the k8s version change.
Another possibility was when the doc was first written it was using minikube which defaults to 8443.

Reply:
I can't really seem to find links to older version of the Kubernetes docs on the matter. I'd guess the docs probably did use minikube which defaults to 8443 rather than somebody configuring `kubeadm` to use port 8443 over 6443. Should I make a note of this in the docs?

Reply:
Feel free to add to the notes below sth like `The port for the Kubernetes API server may be different depending on your Kubernetes configuration.` Does that make sense?